internal/sanitize/container.go

```diff
@@ -55,6 +57,27 @@ func (c *Container) checkImageTags(ctx context.Context, image string) {
 	}
 }
 
+func (c *Container) checkImageRegistry(ctx context.Context, image string) {
+	tokens := strings.Split(image, "/")
```

Comment:
Move this where its used ie if no registries are defined -> no need to split

Reply:
I've moved the check for registries outside, so this is always needed